modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id=using-cluster-logging-to-find-Knative-Serving-logs_{context}"]
+= Using Cluster Logging to find Knative Serving logs
+
+Knative Serving provides default logs for your application. 
+
+.Prerequisites
+
+* xref:../efk-logging-about_{context}[Cluster Logging components] are installed.
+
+.Procedure
```

Comment:
Most of this procedure simply covers "how to get to kibana" and should be there only once I think? Below's file has an exact duplicate.